services/permits/app/compare_extraction_results.py

```diff
@@ -193,6 +214,8 @@ def validate_condition(csv_pairs):
             key=lambda c: c.get("Key"),
         )
 
+        print([c.get("Key") for c in match_results["context_comparison_results"]])
```

Comment:
this log looks more like it was left in by mistake?

Reply:
Definitely, removed it.